README.md

```diff
@@ -19,6 +19,7 @@ $ ./start.py compose-env \
     # Have a look if it looks right
 $ eval $(./start.py compose-env --firmware-file-storage-dir path_to_fw_data_dir)
 $ export FACT_DOCKER_POSTGRES_PASSWORD=mypassword
+$ docker volume create postgres_data
```

Comment:
nit: This could should be named more fact specific (fact_postgres_data?). Shouldn't it?